docker-compose.yml

```diff
@@ -21,22 +36,51 @@ services:
     volumes:
       - ./data/db:/data
       - ./data/db/virtuoso.ini:/data/virtuoso.ini
-  migrations:
+    labels:
+      - "logging=true"
     restart: always
+    logging: *default-logging
+  migrations:
     image: semtech/mu-migrations-service
     links:
       - db:database
     volumes:
       - ./config/migrations:/data/migrations
+    labels:
+      - "logging=true"
+    restart: always
+    logging: *default-logging
   resource:
     image: semtech/mu-cl-resources:1.20.0
     links:
       - db:database
     volumes:
       - ./config/resources:/config
+    labels:
+      - "logging=true"
+    restart: always
+    logging: *default-logging
   cache:
     image: semtech/mu-cache:2.0.1
     links:
       - resource:backend
+    labels:
+      - "logging=true"
+    restart: always
+    logging: *default-logging
+  file:
+    image: semtech/mu-file-service:3.1.1
```

Comment:
It seems that 3.1.2 is the latest version. Is there a reason why that one can't be used?

Reply:
using the same version as loket.

Reply:
I would go for the latest since that seems to include bugfixes. I'm not sure why we need to keep this in sync with another project.

Reply:
okay updated to 3.1.2